src/blocks/scratch3_control.js

```diff
@@ -1,19 +1,19 @@
 var Cast = require('../util/cast');
 var Timer = require('../util/timer');
 
-function Scratch3ControlBlocks(runtime) {
+var Scratch3ControlBlocks = function (runtime) {
```

Comment:
For a circular reference like this, you would need make an exception for the `no-use-before-define` rule, but the function definition style wouldn't make any difference since by the time the function was called, it would be defined.